services/dhcp/src/main/java/org/openthinclient/service/dhcp/RemotedBean.java

```diff
@@ -28,19 +31,20 @@
 	private static final Logger logger = LoggerFactory.getLogger(RemotedBean.class);
 
 	public boolean dhcpReloadRealms() throws Exception {
-//		final ObjectName objectName = new ObjectName("tcat:service=ConfigService");
-//		final MBeanServer server = MBeanServerFactory.findMBeanServer(null).get(0);
-//
-//		if (Boolean.FALSE.equals(server.invoke(objectName, "reloadRealms",
-//				new Object[]{}, new String[]{}))) {
-//			logger.error("Unable to reloadRealms");
-//			return false;
-//		} else
-//			return true;
+		final ObjectName objectName = new ObjectName("tcat:service=ConfigService");
```

Comment:
As we're relying on the application wiring on spring, there is no need at all to use JMX to communicate with the DhcpService. Instead, inject the DhcpService as a constructor argument into RemotedBean, assign it to a local final field, and call ```reloadRealms``` without any JMX or reflection magic at all.

This will reduce the size of the class and make the code more easily maintainable